src/lib/atoms/HelperText/HelperText.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { cloneElement, FC, JSX, useRef } from 'react';
+import { ErrorAlertFill, WarningFill } from '@geneui/icons';
+import classnames from 'classnames';
+
+//Hooks
+import { useEllipsisDetection } from '../../../hooks';
+
+//Components
+import Tooltip from '../../molecules/Tooltip';
+
+// Styles
+import './HelperText.scss';
+
+interface IHelperTextProps {
+    /**
+     * Defines the size of the helper text.<br>
+     * Possible values: `medium | small`.
+     */
+    size?: 'medium' | 'small';
+    /**
+     * Specifies the type of the helper text. <br>
+     * Possible values: `rest | danger | warning`.
+     * `rest` for default information, `danger` for error messages, or `warning` for cautions.
+     */
+    type?: 'rest' | 'danger' | 'warning';
+    /**
+     * The actual text content to be displayed as helper text.
+     * This provides guidance or additional information related to the input field.
+     */
+    text: string;
+    /**
+     * Optional. Icon to be displayed alongside the helper text.
+     * If not provided, a default icon will be used based on the `type` prop.
+     */
+    Icon?: JSX.Element;
```

Comment:
Here in the description, we also need to mention behavior when a dev passes the type and Icon prop combination